### PR TITLE
feat: Add Thesaurus Historische Persoonsgegevens

### PR DESCRIPTION
**Pull request by ddeboer**
* Re-use Poolparty query and adjust for `skos:definition`,
  used by PiCoT.

### CODE REVIEW

File: packages/network-of-terms-catalog/catalog/datasets/picot.jsonld [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  "name": [
+    {
+      "@language": "nl",
+      "@value": "Thesaurus Historische Persoonsgegevens"
+    }
+  ],
+  "alternateName": [
+    {
+      "@language": "en",
+      "@value": "PiCoT"
```
Comment (ddeboer) [inline packages/network-of-terms-catalog/catalog/datasets/picot.jsonld:14]: @RubenSchalk @pwoltjer This is the (common) abbreviation for this source; cf. NTA, GTAA etc. I changed this from `PiCo` to `PicOT` to differentiate the _ontology_ from the _terms list_. I saw @pwoltjer (👋) using `picot:` in the [examples](https://github.com/CBG-Centrum-voor-familiegeschiedenis/PiCo/blob/main/examples/extending-pico/slavenregister.ttl#L7), so `PiCoT` seems to make sense. Agreed?

File: packages/network-of-terms-catalog/catalog/datasets/picot.jsonld [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  "url": [
+    "https://terms.personsincontext.org/ThesaurusHistorischePersoonsgegevens/"
+  ],
+  "mainEntityOfPage": [
+    "https://terms.personsincontext.org/ThesaurusHistorischePersoonsgegevens"
+  ],
+  "description": [
+    {
+      "@language": "nl",
+      "@value": "Rollen voor het beschrijven van personen in historische documenten"
```
Comment (ddeboer) [inline packages/network-of-terms-catalog/catalog/datasets/picot.jsonld:36]: @rschalkrce @pwoltjer Not just roles but also source types, though. Can you provide a better description?

Comment (rschalkrce) [reply packages/network-of-terms-catalog/catalog/datasets/picot.jsonld:36]: I will pass this on to Pieter

**Pull request merged**